src/service/rest.js

```diff
@@ -0,0 +1,58 @@
+const get = async (path) => {
+    await wait();
```

Comment:
You are already prepared to ask for a month of time for performance improvements ^^.